Sources/MessagingInApp/Provider/InAppProvider.swift

```diff
@@ -4,16 +4,16 @@ import Gist
 
 // wrapper around Gist SDK to make it mockable
 internal protocol InAppProvider: AutoMockable {
-    func initialize(organizationId: String, delegate: GistDelegate)
+    func initialize(siteId: String, region: Region, delegate: GistDelegate)
```

Comment:
Two reasons that came into my mind for suggesting this:

- Changing `Region` would break `InAppProvider`  specially when we are going to expose `InAppProvider` to the world
- Makes testing easier. 

Either way, not a blocker.